packages/plugin/src/__tests__/ChartPlugin.spec.js

```diff
@@ -108,12 +108,13 @@ describe('ChartPlugin', () => {
         ['option2', { defaultValue: null }],
     ]
     let props
-    let shallowChartPlugin
+    let chartPlugin
     const canvas = () => {
-        if (!shallowChartPlugin) {
-            shallowChartPlugin = shallow(<ChartPlugin {...props} />)
+        if (!chartPlugin) {
+            chartPlugin = mount(<ChartPlugin {...props} />)
```

Comment:
This is necessary because `jest` and `enzyme` [don't run `useEffect` hooks on a shallow mount](https://airbnb.io/enzyme/#react-hooks-support)